src/test/a11y/a11y.ts

```diff
@@ -73,5 +73,4 @@ describe('Accessibility', () => {
   checkPaths(ClaimIssueErrorPaths)
   checkPaths(DefendantFirstContactPaths)
   checkPaths(DefendantFirstContactErrorPaths)
-  checkPaths(DefendantResponsePaths)
```

Comment:
Why has it been removed? Response journey needs to be covered by a11y tests.

Reply:
accidentally. I'll revert it